packages/amplify-e2e-core/src/categories/geo.ts

```diff
@@ -395,6 +421,25 @@ export function removeFirstDefaultGeofenceCollection(cwd: string): Promise<void>
     .runAsync()
 }
 
+/**
+ * Remove an existing default device tracker. Assume auth is already configured and three device trackers added with first default
+ * @param cwd command directory
+ */
+ export function removeFirstDefaultDeviceTracker(cwd: string): Promise<void> {
+  return spawn(getCLIPath(), ['geo', 'remove'], { cwd, stripColors: true })
+    .wait('Select which capability you want to remove:')
+    .sendKeyDown(3)
+    .sendCarriageReturn()
+    .wait('Select the device tracker you want to remove')
+    .sendCarriageReturn()
+    .wait('Are you sure you want to delete the resource?')
+    .sendConfirmYes()
```

Comment:
assuming this prompt is coming from the prompter library, the E2E test should use `sendYes()` instead of `sendConfirmYes()`. `sendConfirmYes` sends an extra newline (which is only required when using inquirer) that can lead to race conditions with the next prompt in the chain

Reply:
Looking into this, on geo we seem to use `removeResource` from `amplify-cli` which uses `promptConfirmationRemove` from `amplify-cli-core`. This function seems to use `confirmPrompt` which requires the enter key press for confirmation?

Reply:
No that's fine if this prompt is coming from a prompt that hasn't been migrated